bench/polyline_bench.exs

```diff
@@ -0,0 +1,42 @@
+defmodule PolylineBench do
+  use Benchfella
+
+  @medium_polyline "i|~wAeo{aVw@i@SI]EkN^c@@KfXGNULcCo@}HgByEkAcFcAsCk@oAYeAYgZuGiBu@wCi@iGo@eKBiHx@aGzAeMpEgJ`Dy@wC~@kK|D_A`@yLlEkAXuJhDuAj@yAp@mKzD{h@bRu@NcIpCmIbDmGxBk@RkD`AgBj@wAf@a@mBe@sCiCiNkCcMgCkMeBZWE}@BmKsAkCWwE]{BGyC?iBD}BJwCVgDb@mByNu@wSGaC{DL"
+  @medium_linestring @medium_polyline |> Polyline.decode()
+
+  @long_polyline Path.join([".", "test", "fixtures", "long.polyline.txt"])
```

Comment:
The polyline I tested had 160k bytes considerably larger than the given in the test. 

Notice that the larger the polyline is the highest the improvement of the decode function will be because the complexity is quadratic. I didn't want to add it because as you noticed you can see the improvement with the existing one in the fixtures.

Reply:
@carrascoacd if you can share it, I'll add an "extra-large" version 😄

Reply:
Alright! let me share it

[results-20200826-140926.txt](https://github.com/pkinney/polyline_ex/files/5142194/results-20200826-140926.txt)